src/mongocxx/test/v_noabi/change_streams.cpp

```diff
@@ -457,20 +457,17 @@ TEST_CASE("Documentation Examples", "[change_stream]") {
     }};
 
     SECTION("Example 1") {
```

Comment:
>  the code itself remains to preserve code coverage as part of the test suite.

Suggest removing the doc examples from tests to avoid going out-of-sync. I expect the main purpose is documentation rather than testing. And the examples are [run on most Evergreen configurations](https://github.com/mongodb/mongo-cxx-driver/blob/079af6c4a04f67c8f96cb100a77cf1e25eeabe9e/.evergreen/scripts/test.sh#L314-L320).